test/certs/invalid-cert.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN TRUSTED CERTIFICATE-----
```

Comment:
Also this addition belongs to the main commit?

Reply:
Oops; fixed as well.